docs/rules/use-ember-get-and-set.md

```diff
@@ -12,6 +12,8 @@ ember/use-ember-get-and-set: [2, {
 
 Setting `ignoreThisExpressions` to `true` allows use of `this.get()` and `this.set()` where you will generally know if `this` is an `Ember.Object`.
 
+Setting `onlyLintThisExpressions` to `true` allows use of non-Ember objects like `server.get()` and `map.set()`.
```

Comment:
Perhaps we should rename to `ignoreNonThisExpressions` continue the pattern of each option being used to ignore a different type of object?

Reply:
Done